source/shared/cpp/ObjectModel/Enums.h

```diff
@@ -357,4 +357,38 @@ GenerateStringToEnumMap(const std::unordered_map<T, std::string, EnumHash>& keyT
     }
     return result;
 }
+
+void InitializeAdaptiveCardSchemaKeyEnumMappings();
```

Comment:
>InitializeAdaptiveCardSchemaKeyEnumMappings [](start = 5, length = 43)

We probably don't need these here in the header as they're not called outside of Enums.cpp